exceptions/exception_examples.py

```diff
@@ -0,0 +1,108 @@
+"""Handling exceptions
+   - catch multiples exceptions
+   - catch exceptions with arguments
+   - re-raising exception to get stack"""
```

Comment:
there's no example of custom exception inheritance. Imagine you have an app and the base app exception. you can inherit from that exception and catch only base one instead of all other.

Reply:
there's also no example of `try/else`

Reply:
Now in my TODOs. I'll push it in the next PR.